deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml

```diff
@@ -105,7 +105,16 @@ spec:
     Hat Marketplace products, you will need to make the pull secret available across
     the cluster. This can be achieved by applying the Red Hat Marketplace Pull Secret
     as a [global pull secret](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets).
-    For alternative approachs, please see the official OpenShift [documentation](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).\n\n\n###
+    For alternative approachs, please see the official OpenShift [documentation](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).\n\n###
+    Subscription Config\n\nIt is possible to [configure](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md)
+    how OLM deploys an Operator via the `config` field in the [Subscription](https://github.com/operator-framework/olm-book/blob/master/docs/subscriptions.md)
+    object.\n\nThe Red Hat Marketplace Deployment Operator will also read the `config`
+    and append it to the operands. The primary use case is to control scheduling using
+    [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+    and [NodeSelectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).\n\nA
+    limitation is that the `config` elements are only appended to the operands. The
+    elements in the operands are not removed if the `config` is removed from the `Subscripion`.
+    The operand must be modify manually, or deleted and recreated by the controller.\n\n###
```

Comment:
`The operand must be modify manually`
modified*

Reply:
corrected